tst/EnergyPlus/unit/OutputReportTabularAnnual.unit.cc

```diff
@@ -219,3 +219,68 @@ TEST_F( EnergyPlusFixture, OutputReportTabularAnnual_GatherResults )
 
 
 
+TEST_F(EnergyPlusFixture, OutputReportTabularAnnual_columnHeadersToTitleCase)
+{
+	std::string const idf_objects = delimited_string ({
+		"Version,8.6;",
+		"Output:Table:Annual,",
+		"Test Report, !- Name",
+		", !- Filter",
+		", !- Schedule Name",
+		"OnPeakTime, !- Variable or Meter 1 Name",
+		"HoursNonZero, !- Aggregation Type for Variable or Meter 1",
+		"0, !- field Digits After Decimal 1",
+		"Electricity:Facility, !- Variable or Meter 2 Name",
+		"SumOrAverageDuringHoursShown, !- Aggregation Type for Variable or Meter 2",
+		", !- field Digits After Decimal 2",
+		"Misc Facility Electric Energy, !- Variable or Meter 3 Name",
+		"SumOrAverage, !- Aggregation Type for Variable or Meter 3",
+		"0; !- field Digits After Decimal 3",
+		"",
+		"Schedule:Compact,",
+		"    OnPeakTime,              !- Name",
+		"    Fraction,                !- Schedule Type Limits Name",
+		"    Through: 12/31,          !- Field 1",
+		"    For: Weekdays SummerDesignDay,  !- Field 2",
+		"    Until: 12:00, 0.0,       !- Field 4",
+		"    Until: 20:00, 1.0,       !- Field 6",
+		"    Until: 24:00, 0.0,       !- Field 8",
+		"    For: AllOtherDays,       !- Field 9",
+		"    Until: 24:00, 0.0;       !- Field 11",
+	});
+
+	ASSERT_FALSE (process_idf (idf_objects));	
+
+	Real64 facilUse;
+	SetupOutputVariable ("Misc Facility Electric Energy [J]", facilUse, "Zone", "Sum", "Lite1", _, "Electricity", "Facility", "General"); //create an electric meter
+
+	OutputProcessor::NumEnergyMeters = 2;
+	OutputProcessor::EnergyMeters.allocate(OutputProcessor::NumEnergyMeters);
+	OutputProcessor::EnergyMeters(1).Name = "Electricity:Facility"; //"ELECTRICITY:FACILITY";
+	OutputProcessor::EnergyMeters(2).Name = "ELECTRICITY:LIGHTING";
+
+	DataGlobals::DoWeathSim = true;
+
+	OutputReportTabularAnnual::GetInputTabularAnnual();
+
+	EXPECT_EQ (OutputReportTabularAnnual::annualTables.size (), 1u);
+
+	std::vector<AnnualTable>::iterator firstTable = OutputReportTabularAnnual::annualTables.begin ();
```

Comment:
Is there a reason to not just get the first table? Doesn't look like you are doing anything below that requires an iterator.

`AnnualTable & firstTable = OutputReportTabularAnnual::annualTables[0];`